src/datasets/dataset_dict.py

```diff
@@ -758,6 +1131,13 @@ def from_csv(
 
         Returns:
             :class:`DatasetDict`
+
+        Example:
+
+        ```py
+        >>> from datasets import DatasetDict
+        >>> ds = DatasetDict.from_csv('path/to/dataset.csv')
```

Comment:
you need to pass a dict here
```suggestion
        >>> ds = DatasetDict.from_csv({'train': 'path/to/dataset.csv'})
```